src/index.js

```diff
@@ -21,9 +22,16 @@ class Node extends libp2p {
         webRTCStar
       ],
       connection: {
-        muxer: [
-          spdy
-        ],
+        muxer: process.env.LIBP2P_MUXER ? (() => {
+          const muxerPrefs = process.env.LIBP2P_MUXER
+          return muxerPrefs.split(',').map((pref) => {
+            switch (pref) {
+              case 'spdy': return spdy
+              case 'multiplex': return multiplex
+              default: throw new Error(pref + ' muxer not available')
+            }
+          })
+        })() : [spdy],
```

Comment:
Right now it means `Default to spdy`, that might change if multiplex becomes the 'muxer', until them, the env variables help for running tests.